ExampleApp/ExampleApp/Common/DataSource/SectionModelItem.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+protocol SectionModelItem {
+
+    var itemsCount: Int { get }
+
+    var sectionName: String? { get }
+
+    var rowData: [Any] { get }
+
+    func cellClass() -> UIView.Type
```

Comment:
Why is this function and not computed property? The other getter-only information is provided by the properties, this one is not

Reply:
Done